src/mount/bsd.rs

```diff
@@ -371,19 +371,27 @@ impl<'a> Nmount<'a> {
     pub fn nmount(&mut self, flags: MntFlags) -> NmountResult {
         // nmount can return extra error information via a "errmsg" return
```

Comment:
This comment got separated from the code it describes.  It should be directly above line 380.